### DISCUSSION TITLE
Can't access the QTable :selected array directly

### DISCUSSION THREAD

**Discussion by HappyGingerWolf** [quasarframework/quasar#12576] [General - Components / Directives / etc]
I can't access the :selected array of a QTable directly, the only way I've found to retrieve it is by using the details returned by the @selection event, but this means i can't select fields through code.

I've tried using it with and without :selected.sync, with v-model:selected, and with various ways of defining the variable for selected, but they never connect. For example, if I use :selected.sync="selectedItems", selectedItems will always stay as it was defined initially, even if rows have been selected in the table.

**Comment by metalsadman** [comment]
`v-model:selected` https://v3-migration.vuejs.org/breaking-changes/v-model.html#overview

**Reply by HappyGingerWolf** [reply]
I did that too, same results

**Reply by MilosPaunovic** [reply]
Provide a Codepen reproduction of your problem, that way you'll get better support.
